backends/qualcomm/builders/op_add.py

```diff
@@ -15,7 +15,7 @@
 
 @register_node_visitor
 class Add(NodeVisitor):
-    target = ["aten.add.Tensor"]
+    target = ["aten.add.Tensor", "aten.add_.Tensor"]
```

Comment:
I feel like `add_` technically is an in-place op, and qualcomm's add operator is not the same